R/DataMart.R

```diff
@@ -484,6 +504,26 @@ getCompanySecret <- function()
     return (secret)
 }
 
+#' Gets document secret from the environment. Throws an error if not found.
+#'
+#' @return Document secret token as a string.
+#'
+#' @noRd
+getProjectSecret <- function()
+{
+    secret <- get0("projectSecret", ifnotfound = "")
```

Comment:
Is this defined in the `.GlobalEnv`? If so, it should be specified with the `envir` arg.